docs/snowpark.md

```
@@ -13,11 +13,11 @@ noContent: false
 
 With the new Snowpark integration in Deepnote you can perform database transformations with Python and deploy machine learning models to Snowflake—**without moving your data or changing the Python code you already use**.
 
-The Snowpark + Deepnote integration makes the warehouse feels like an in-memory object. Simply write your code in Deepnote and manipulate your tables as if you were using Pandas. All compute occurs directly in the warehouse so there’s no need to constantly move your data around.
+The Snowpark + Deepnote integration makes the warehouse feel like an in-memory object. Simply write your code in Deepnote and manipulate your tables as if you were using Pandas. All compute occurs directly in the warehouse so there’s no need to constantly move your data around.
 
 ### How to connect
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)